src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -243,7 +242,7 @@ public static Day parseDay(String dayAsStr) throws ParseException {
     public static LocalTime parseLocalTime(String localTimeAsStr) throws ParseException {
         try {
             int hour = Integer.parseInt(localTimeAsStr);
-            if (!Arrays.asList(Timetable.START_TIMINGS).contains(hour)) {
+            if (hour < 0 || hour > 23) {
```

Comment:
No "magic numbers" please. Bad practice.

Reply:
Then please, add in these values into the TimeUtils. It’s your issue.

Reply:
Also, using purely the start timings will not work, because your start timings end at 21, but some classes end at 22. As such, I could not stick with the original implementation. Please improve it on your side.

Reply:
So you need 24hrs timetable?

Reply:
No. I need the start timings to end at 22 instead. The start timings I can change back. Please give public static variables for what the start and end times should be.

Reply:
fixed